### PR TITLE
Migrate to Jalpc template

### PR DESCRIPTION
**Pull request by gmrukwa**
Migration has begun, some initial configurations have been
conducted. Sample content has to be removed in short time.

There are problems with displaying code chunks - no
highlighting works.

Page with the blog posts list should also be cleaned up
and formatted properly.

### CODE REVIEW

**Comment by gmrukwa** [comment]
[Tweak github pages with jekyll template](https://trello.com/c/JyoCPgeg/157-tweak-github-pages-with-jekyll-template)

File: _includes/blog_header.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<div class="search-tool"
+      style="position: fixed; top: 0px ; bottom: 0px; left: 0px; right:  0px; opacity: 0.95; background-color: #111111; z-index: 9999; display: none;">
```
Comment (FreexD) [inline _includes/blog_header.html:2]: Use classes to define styles

File: _includes/sections/skills.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            <h1><span data-i18n="skills.my_skills">My Skills</span></h1>
+        </div>
+    </div>
+    <div class="row features-block">
+        <div class="wow zoomIn col-lg-6 col-lg-offset-3">
+            <canvas id="{{ site.data.index.skills.id }}" height="500" width="500"></canvas>
+        </div>
+        <div class="col-lg-1"></div>
+        <script>
+        var ctx = document.getElementById("{{ site.data.index.skills.id }}");
```
Comment (FreexD) [inline _includes/sections/skills.html:14]: Try to stay consistent. Digger use jQuery everywhere od dont use it at all ;)

File: blog/donate.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+layout: blog
+title: Donate
+permalink: /donate/
+---
+    <link href="{{ "/static/css/slick.css" | prepend: site.baseurl }}" rel="stylesheet">
+    <link href="{{ "/static/css/slick-theme.css" | prepend: site.baseurl }}" rel="stylesheet">
+<div class="row">
+    <div class="col-sm-8 col-sm-offset-2 text-center">
+        <div class="navy-line"></div>
+        <h1>Donate Jalpc</h1>
```
Comment (FreexD) [inline blog/donate.html]: Does it need to be here?

**Review by FreexD — COMMENTED** [review]
My suggestions are probably targetted at the template itself, not your work ;) and these changes are of low priority so lgtm. GJ ;)

File: blog/donate.html (hunk 3 above)
Comment (gmrukwa) [reply blog/donate.html]: I missed this file absolutely. Don't know how.

File: _includes/blog_header.html (hunk 1 above)
Comment (gmrukwa) [reply _includes/blog_header.html:2]: This is **the** search tool provided by the template. We may upgrade it in further iterations.

File: _includes/sections/skills.html (hunk 2 above)
Comment (gmrukwa) [reply _includes/sections/skills.html:14]: I'm not sure, whether we will ever use this file. It is just preserved as part of template, but will probably disappear.

**Review by FreexD — APPROVED** [review]
LGTM

**Pull request merged**